### ISSUE TITLE
Bridget: hot-plugging is not working

### ISSUE THREAD

**Issue by Williangalvani** [bluerobotics/BlueOS-docker#1016]
We had a report that upon disconnecting and reconnecting a serial port, the bridge isn't rebuilt

**Comment by patrickelectric** [comment]
The latest version of bridges now has an option to disable auto disconnect and also detect serial disconnection. 
https://github.com/patrickelectric/bridges/commit/16a7b82658823c1b9f119c887be38ea0475eb75e
https://github.com/patrickelectric/bridges/commit/459cf353ad7a7b5ace8fe6f941e8601caff62c87

Will create a PR for blueos soon.

**Comment by rafaellehmkuhl** [comment]
@patrickelectric was this solved with the recent PRs?

**Comment by patrickelectric** [comment]
No, this makes possible for bridget to restart bridges if something goes wrong on the serial port.

**Comment by patrickelectric** [comment]
bridges itself now just closes if a serial port is disconnected.
